### PR TITLE
Metrics app

### PR DESCRIPTION
**Pull request by rolandfbk**
This web app gets the initial data from an API. It is the list of all African countries and its details.

- I created the store from the metricsReducer
- metricsReducer updates the state with the data from the API
- Metrics component displays the list of all countries
- CountryDetails components displays the details of a selected country

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <header>
+      <nav>
+        <div style={{ cursor: 'pointer' }} aria-hidden onClick={() => navigate(-1)}>
+          <span className="leftNav">
+            <FontAwesomeIcon icon={faChevronLeft} />
+            Africa
+          </span>
+        </div>
+        <div>
+          <span className="middleNav">Directory</span>
```
Comment (julie-ify) [inline src/components/Navbar.js]: - [ ] From the design template, **`"Directory"`** is too generic to be your caption for the app. Kindly include something more descriptive of the page contents

File: README.md [hunk 2]
```diff
@@ -1,70 +1,79 @@
...
-### Deployment
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```
Comment (julie-ify) [inline README.md:73]: - [ ] In your Acknowledgments section kindly acknowledge Nelson Sakwa for the design in keeping with the Creative Commons License requirement.

- [ ] During the course of your video, you seem to be inaudible; and you failed to highlight the piece of code, you're most proud of. Kindly re-record the video and highlight the piece of code you find interesting.

File: src/App.test.js [hunk 3]
```diff
@@ -1,8 +1,30 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import Metrics from './components/Metrics';
+import CountryDetails from './components/CountryDetails';
+import store from './redux/ConfigureStore';
```
Comment (julie-ify) [inline src/App.test.js]: - [ ] Kindly include some integration tests using the React Testing Library.

File: src/components/Metrics.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import { React, useEffect } from 'react';
+import { useDispatch, useSelector, shallowEqual } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowAltCircleRight } from '@fortawesome/free-solid-svg-icons';
+import { getDataFromApi } from '../redux/metric-redux';
+
+const Metrics = () => {
```
Comment (julie-ify) [inline src/components/Metrics.js:8]: - [ ] In your Homepage kindly include a filtering mechanism, like a search box that users can use to filter countries on the home page.

  wuye-1

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @rolandfbk,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rolandfbk** [comment]
Hi, my Code Reviewer, I recorded the video more than 10 times, but I'm still getting an audio issue. After this module, I will take the laptop to the technician to fix it. Here I come to ask for your clemency, please don't let me fail this project because of the sound. I don't know where I can get another laptop right now. please

**Review by elyor-doniyorov — APPROVED** [review]
Hi @rolandfbk, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 
- [OPTIONAL] Kindly update your video introduction link when you fix your laptop issue.

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by rolandfbk** [comment]
Thank you @elyor-doniyorov, I will fix the laptop this weekend

**Pull request merged**